_includes/head.html

```diff
@@ -12,6 +12,9 @@
   <link rel="canonical" href="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title }}" href="{{ "/feed.xml" | prepend: site.baseurl | prepend: site.url }}" />
 
+  <link rel="shortcut icon" href="{{ "/images/favicon.ico" | prepend: site.baseurl }}" type="image/x-icon">
+  <link rel="icon" href="{{ "/images/favicon.ico" | prepend: site.baseurl }}" type="image/x-icon">
```

Comment:
On a related note, I couldn't figure out how to tell Jekyll to pick up files off the root ('they' recommend putting favicon.ico in the root of the site).....which is why I put it in images.  @thejameskyle do you know?